docker-compose/docker-compose.yml

```diff
@@ -14,7 +12,7 @@ version: "2"
 
 services:
     alfresco:
-        image: alfresco/alfresco-content-repository:6.2.0-A3
+        image: quay.io/alfresco/alfresco-content-repository:6.2.0-A5
```

Comment:
The tag is available on docker hub:
https://hub.docker.com/r/alfresco/alfresco-content-repository/tags